packages/underscore/package.js

```diff
@@ -0,0 +1,42 @@
+
+Package.describe({
+  summary: "JavaScript's utility _ belt",
+  version: '1.9.2',
```

Comment:
maybe this PR isnt relevant anymore, but if it is, why do we decreasing the version here?